lib/user_agent/browsers/webkit.rb

```diff
@@ -2,12 +2,12 @@ class UserAgent
   module Browsers
     class Webkit < Base
       def self.extend?(agent)
-        agent.detect { |useragent| useragent.product == 'AppleWebKit' }
+        agent.detect { |useragent| useragent.product =~ /\AAppleWebKit\z/i }
       end
 
       def browser
         if os =~ /Android/
-          'Android'
+          'Browser'
```

Comment:
I would rather leave this as `Android` as people may well have code relying on this return value. As the full name seems to be "Android Browser" it being shortened to "Android" seems as valid, and perhaps more clear in this context, than shortening it to "Browser".

What do you think?